keyform/forms.py

```diff
@@ -15,7 +15,7 @@ class Meta:
     def __init__(self, *args, **kwargs):
         super(CreateForm, self).__init__(*args, **kwargs)
         # removes blank choices from Radio Select options
-        self.fields['payment_method'] = TypedChoiceField(widget=RadioSelect(), choices=Request.PAYMENT_TYPES)
+        self.fields['payment_method'] = TypedChoiceField(widget=RadioSelect(), choices=[('', _('Not Applicable'))] + list(Request.PAYMENT_TYPES), required=False)

```

Comment:
Why wasn't 'Not Applicable' added to the models instead of being added here?
I don't see why required is set to false here either.